0-ad-history-mod-package/simulation/templates/gaia/fauna_bear_brown.xml

```diff
@@ -24,6 +24,10 @@
     <GenericName>Brown Bear</GenericName>
     <SpecificName>Ursus arctos</SpecificName>
     <Icon>gaia/fauna_bear_brown.png</Icon>
+    <History>
+        The Brown Bear has the widest distribution of the bear species, and it ranges from North America to Europe to Asia. Due to its widespread habitat there are many subspecies of Urs Arctos, some which look quite different.
+	Brown bears have dense fur coat that can not only be brown but also blond or black. They are best recognized by the large hump on their shoulders. They can grow up to nine feet long and can weigh 550 pounds. While it usually eats plants a Brown Bear would not hesitate to eat meat and of all the species of bear the brown bear is probably the most qualified to take it due to its size and power. Brown Bears hibernate during the winter to avoid the dearth of food during that time.
```

Comment:
```suggestion
	Brown bears have dense fur coats that can not only be brown but also blond or black. They are best recognized by the large hump on their shoulders. They can grow up to nine feet long and can weigh 550 pounds. While it usually eats plants, a Brown Bear would not hesitate to eat meat, and of all the species of bear the brown bear is probably the most qualified to take it due to its size and power. Brown Bears hibernate during the winter to avoid the dearth of food during that time.
```